shimmer-hornet/docker-compose.yml

```diff
@@ -46,10 +46,11 @@ services:
   ##################################################################
 
   traefik:
-    image: traefik:v2.8.1
+    image: traefik:latest
```

Comment:
Could lead to problems if configuration changes. I would prefer a fixed version number.

Reply:
See it also like bogenlos

Reply:
would be v2.8 better? 
Think this should be latest v2.8.x then
https://hub.docker.com/_/traefik

Reply:
Same as by other nodes would be good. That we have one standard.

Reply:
reverted this to 2.8.1 for this pr